storybookjs/storybook
# Angular component props assignment improvements

Issue:
Components that use ngModel directive and implement ControlValueAccessor interface are not supported.
Angular doesn't call ngOnChnages hook in dynamically created components [issue](https://github.com/angular/angular/issues/8903)  

## What I did
- enabled 'noImplicitAny' flag in tsconfig.json
- implemented manual call of "ngOnChanges" livecycle hook
- implemented ngModel property support & added FormModule import to module
- fixed overriding of components default values with "undefined" value
- added "types/lodash" dependency
- added example story
- minor typing and tslint fixes

## How to test
Run example/angular. "ngModel" section illustrates how to define value to components with ngModel

Is this testable with jest or storyshots?
No

Does this need a new example in the kitchen sink apps?
Yes. Added

Does this need an update to the documentation?
Yes, but it angular specific and i didn't find corresponding section in documentation

If your answer is yes to any of these, please make sure to include it in your PR.
